CONTRIBUTING.md

```diff
@@ -164,6 +165,27 @@ The footer should contain any information about **Breaking changes** and is also
 
 **Breaking Changes** should start with the word `BREAKING CHANGE:` with a space or two newlines. The rest of the commit message is then used for this.
 
+### Submitting a Pull Request
+
+Prior to developing a new feature, be sure to search the [Pull Requests](https://github.com/ciscospark/spark-js-sdk/pulls) for your idea to ensure you're not creating a duplicate change. Then, create a development branch in your forked repository for your idea and start coding!
+
+When you're ready to submit your change, first check that new commits haven't been made in the upstream's `master` branch. If there are new commits, rebase your development branch to ensure a fast-forward merge when your Pull Request is approved:
+
+```bash
+# Fetch upstream master and update your local master branch
+git fetch upstream
+git checkout master
+git merge upstream/master
+
+# Rebase your development branch
+git checkout feature
+git rebase master
+```
+
+Make sure all tests pass with your changes. See [Running Tests](#running-tests) above for details about running tests.
```

Comment:
We should probably remove this line until we're setup for external developers to create test users (or at least, encourage them to write unit tests and make sure those pass).